lib/wallet/http.js

```
@@ -1857,22 +1878,24 @@ class TransactionOptions {
    * @constructor
    * @param {RequestValidator} [valid]
    * @param {(NetworkType|Network)?} [network]
+   * @param {AbortSignal} [signal]
    */
 
-  constructor(valid, network) {
+  constructor(valid, network, signal) {
     if (valid)
-      return this.fromValidator(valid, network);
+      return this.fromValidator(valid, network, signal);
```

Biome (2.1.2)

[error] The constructor should not return a value.

The constructor is here:

Returning a value from a constructor may confuse users of the class.

(lint/correctness/noConstructorReturn)